view.html

```diff
@@ -26,25 +22,14 @@
 
 <body>
     <main>
-
-        <head>
-            <link rel="stylesheet" type="text/css" href="css/styles.css">
-            <meta charset="utf-8" />
-
-        </head>
-
-        <body>
             <div class="sketch">
                 <script src="app/sketch.js"></script>
-                <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.3.1/p5.js"></script>
-                <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.3.1/addons/p5.sound.min.js"></script>
             </div>
-        </body>
     </main>
-
-    <footer class="footer">
-        <h4>view this project on <a href="https://github.com/tkolkmeyer/intention"> github</a></h4>
-    </footer>
 </body>
 
+<footer class="footer">
+    <h4> <a href="https://github.com/tkolkmeyer/intention"> view this project on github</a></h4>
+</footer>
+
```

Comment:
Nah footer goes inside body!!! https://www.geeksforgeeks.org/html5-footer-tag/ 
otherwise very nice changes

Reply:
oh lol, you're right.

I never use the footer tag so I forgot